server/db/migrations/create-admins-table.js

```diff
@@ -0,0 +1,27 @@
+const tableName = 'admins'
+
+module.exports = {
+  async up(knex) {
+    await knex.schema.createTable(tableName, (table) => {
+      // Auto-incrementing non-nullable unsigned integer primary key "id" field
+      table.increments()
+
+      // Simple fields
+      table.string('first_name').notNullable()
+      table.string('last_name').notNullable()
+      table.string('email').notNullable()
+      table.string('password').notNullable()
```

Comment:
This will create a password column with the type `varchar`. I believe that we want to create the column with a `text` type, so we will need `table.text('password').notNullable()`. That will prevent possible issues with encryption length later on.